### ISSUE TITLE
Create zprofile when zsh is installed via wlinux-setup

### ISSUE THREAD

**Issue by lesharris** [WhitewaterFoundry/WLinux#239]
**Is your feature request related to a problem? Please describe.**
Zsh does not source /etc/profile, .profile, except in sh or ksh emulation mode.  This means much of the useful default environment configuration done by Wlinux is not available 'out of the box'

**Describe the solution you'd like**
When zsh is installed via wlinux-setup, also create a /etc/zprofile that either sources the existing /etc/profile or has the equivalent configuration.

**Describe alternatives you've considered**
It is possible to workaround this by adding something like the following to ~/.zprofile:

```[[ -e /etc/profile ]] && emulate sh -c 'source /etc/profile'```

**Additional context**
While it is not a difficult issue to workaround it would be nice to have it done out of the box since zsh is offered as a supported shell in wlinux-setup.

**Comment by sirredbeard** [comment]
We could follow the Arch example and have /etc/zsh/zprofile simply source /etc/profile.

**Comment by grufwub** [comment]
Been working on that here: https://github.com/grufwub/WLinux/tree/zsh-profile
If you check the code comments I added --> I had to work around an issue that appeared down to bash/zsh code not working exactly the same. Whilst my current work-around is good as a permanent fix, there might be another option we can try should this issue appear with other shells too. Also had to change the method being used to set /etc/zsh/zshrc, to appending with tee as previous method wasn't working.

Have tried a fresh build and run-through of wlinux-setup with no issues. Would appreciate anyone else able to give this a test with fresh builds too!
